ow_plexil/src/plexil-adapter/OwlatInterface.h

```diff
@@ -12,17 +12,70 @@
 #include <ros/ros.h>
 
 // ow_plexil
+#include <Value.hh>
 #include "PlexilInterface.h"
 
 // OWLAT Sim (installation required)
 #include <owlat_sim_msgs/ARM_UNSTOWAction.h>
 #include <owlat_sim_msgs/ARM_STOWAction.h>
+#include <owlat_sim_msgs/ARM_MOVE_CARTESIANAction.h>
+#include <owlat_sim_msgs/ARM_MOVE_CARTESIAN_GUARDEDAction.h>
+#include <owlat_sim_msgs/ARM_MOVE_JOINTAction.h>
+#include <owlat_sim_msgs/ARM_MOVE_JOINTSAction.h>
+#include <owlat_sim_msgs/ARM_MOVE_JOINTS_GUARDEDAction.h>
+#include <owlat_sim_msgs/ARM_PLACE_TOOLAction.h>
+#include <owlat_sim_msgs/ARM_SET_TOOLAction.h>
+#include <owlat_sim_msgs/ARM_STOPAction.h>
+#include <owlat_sim_msgs/ARM_TARE_FSAction.h>
+#include <owlat_sim_msgs/TASK_DROPOFFAction.h>
+#include <owlat_sim_msgs/TASK_PSPAction.h>
+#include <owlat_sim_msgs/TASK_SCOOPAction.h>
+#include <owlat_sim_msgs/TASK_SHEAR_BEVAMETERAction.h>
+#include <owlat_sim_msgs/ARM_JOINT_ANGLES.h>
+#include <owlat_sim_msgs/ARM_JOINT_ACCELERATIONS.h>
+#include <owlat_sim_msgs/ARM_JOINT_TORQUES.h>
+#include <owlat_sim_msgs/ARM_JOINT_VELOCITIES.h>
+#include <owlat_sim_msgs/ARM_FT_TORQUE.h>
+#include <owlat_sim_msgs/ARM_FT_FORCE.h>
+#include <owlat_sim_msgs/ARM_POSE.h>
+#include <owlat_sim_msgs/ARM_TOOL.h>
+
+using std::string;
+using std::vector;
+using std::unique_ptr;
+using std::make_unique;
+using namespace PLEXIL;
```

Comment:
Header files should not have 'using namespace',  only `using` for specific symbols.   (This line might have been my bad).  Move this to the .cpp file, and then qualify any symbols needed in this .h file, e.g. PLEXIL::Value.